### ISSUE TITLE
Travis CI build failing

### ISSUE THREAD

**Issue by abishekvashok** [abishekvashok/cmatrix#28]
log:

``` 
make

CDPATH="${ZSH_VERSION+.}:" && cd . && aclocal-1.15 

/bin/bash: aclocal-1.15: command not found

make: *** [aclocal.m4] Error 127
```

Reason: Git doesnot preserve timestamps.

How to overcome this:
`touch` the files.

**Comment by abishekvashok** [comment]
Leaving it for a beginner to claim.

**Comment by abishekvashok** [comment]
Closed in #29
